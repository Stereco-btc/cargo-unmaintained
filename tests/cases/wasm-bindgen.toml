url = "https://github.com/rustwasm/wasm-bindgen"
rev = "0.2.87"
