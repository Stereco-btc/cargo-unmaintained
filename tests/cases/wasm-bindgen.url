https://github.com/rustwasm/wasm-bindgen
