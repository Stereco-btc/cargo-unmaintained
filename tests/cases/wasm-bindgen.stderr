warning: failed to clone `https://hg.mozilla.org/mozilla-central/file/tip/testing/mozbase/rust/mozrunner`
